ui-participant/src/hub/HubPage.tsx

```diff
@@ -1,24 +1,43 @@
-import React, { useState } from 'react'
+import React, { useEffect, useState } from 'react'
 import { usePortalEnv } from '../providers/PortalProvider'
 import { useUser } from '../providers/UserProvider'
 
-import { Enrollee, ParticipantTask, Portal, Study } from '../api/api'
+import Api, { Enrollee, ParticipantTask, Portal, Study } from '../api/api'
 import TaskLink, { getTaskPath, isTaskAccessible, isTaskActive } from './TaskLink'
 import { Link, NavLink } from 'react-router-dom'
 import { DocumentTitle } from 'util/DocumentTitle'
 import { userHasJoinedPortalStudy } from 'util/enrolleeUtils'
 
-import { HubMessageAlert, useHubUpdate } from './hubUpdates'
+import { HubMessageAlert, HubUpdateMessage, useHubUpdate } from './hubUpdates'
 import { filterUnjoinableStudies } from '../Navbar'
+import { ParticipantDashboardAlert, alertDefaults } from '@juniper/ui-core'
 
 
 /** renders the logged-in hub page */
 export default function HubPage() {
-  const { portal } = usePortalEnv()
+  const { portal, portalEnv } = usePortalEnv()
   const { enrollees } = useUser()
+  const [noActivitiesAlert, setNoActivitiesAlert] = useState<ParticipantDashboardAlert>()
+
+  useEffect(() => {
+    loadDashboardAlerts()
+  }, [])
+
+  const loadDashboardAlerts = async () => {
```

Comment:
I like what you did here of loading the dashboard config separately from the big study/portal env blob that gets loaded when they hit the landing page